cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -72,12 +72,12 @@ define([
             );
         });
 
-        it('Changing course start date without active certificate configuration should result in error', function () {
-            this.view.$el.find('#course-start-date')
+        it('Changing course start date without active certificate configuration should not result in error',
```

Comment:
@asadiqbal08 yeah i think we should remove the test here it does not make sense since error message does not exist in code base.